main.css

```diff
@@ -1,3 +1,83 @@
-body {
-  background-color: aliceblue;
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+
+nav > p {
+  left: 12px;
+  width: 90px;
+  height: 28px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 28px;
+  color: #28352f;
+}
+
+nav > a {
+  margin: auto 0;
+}
+
+.headline {
+  background-image: url(img/header-illsutration-mobile.svg);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header > h2 {
+  width: 327px;
+  left: 24px;
+  top: 64px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 58px 0;
+}
```

Comment:
- [ ] Good job with following most of the styles from your chosen template on Figma. Please consider adding the required font-family `Inter` and font-weight `700` so that your design will match that of your chosen template 😃.